airflow/www/static/js/dags.js

```diff
@@ -116,21 +116,27 @@ $.each($("[id^=toggle]"), function toggleId() {
   });
 });
 
-$(".typeahead").typeahead({
-  source(query, callback) {
-    return $.ajax(autocompleteUrl, {
+// eslint-disable-next-line no-underscore-dangle
```

Comment:
```suggestion
```

Not needed anymore

Reply:
I removed it but got linting error, so added it again